internal/it/util.go

```diff
@@ -52,12 +54,17 @@ const (
 	EnvHzVersion          = "HZ_VERSION"
 )
 
-const DefaultPort = 7701
 const DefaultClusterName = "integration-test"
 
-var rc *RemoteControllerClient
+var rc *RemoteControllerClientWrapper
 var rcMu = &sync.RWMutex{}
-var defaultTestCluster *TestCluster
+var defaultTestCluster = NewSingletonTestCluster("default", func() *TestCluster {
+	port := NextPort()
+	if SSLEnabled() {
+		return rc.startNewCluster(MemberCount(), xmlSSLConfig(DefaultClusterName, port), port)
+	}
+	return rc.startNewCluster(MemberCount(), xmlConfig(DefaultClusterName, port), port)
```

Comment:
I couldn't see the shutting down action of spawned cluster. Am I missing somewhere ?

Reply:
That's right, it doesn't get shutdown since it's like a global cluster. I've added a "shutdown cluster after some inactivity" task, but that doesn't work well and introduced more problems.